### DISCUSSION TITLE
Standalone CLI program to read memory timings for Ryzen(or intel)

### DISCUSSION THREAD

**Discussion by Connor-GH** [cyring/CoreFreq#315] [Ideas]
The program ZenTimings on windows is similar to this, and I can't read C very well, much less the tens of thousands of lines, so I was wondering if you had plans to package just the Memory (controller) part.

**Comment by cyring** [comment]
Among my sandbox programs:
* [zencli.c](https://github.com/cyring/Tips/blob/master/C/zencli.c)  

IMC decoding needs low level.
I believe the Linux kernel would be the best place to maintain such code; which would export Timings data through the standard sys file-system attributes for any foreign languages purposes.

**Comment by cyring** [comment]
GoTo #318
